src/scripts/main.js

```diff
@@ -1,3 +1,194 @@
 'use strict';
 
-// write code here
+const tableHeaders = document.querySelectorAll('th');
+const rowsArray = Array.from(document.querySelectorAll('tbody tr'));
```

Comment:
You are querying for the same elements twice (lines 3 and 36). It's better to do it once and reuse the result. This will improve the performance of your code.